data/avgi/avgi weapons.txt

```diff
@@ -0,0 +1,802 @@
+	
+
+outfit "Speckle Turret"
+	category "Turrets"
+	cost 40000
+	thumbnail "outfit/speckle turret"
+	"mass" 16.8
+	"outfit space" -24
+	"weapon capacity" -24
+	"energy capacity" 180
+	"turret mounts" -1
+	"speck capacity" 2400
+	weapon
+		sound "fleck"
+		ammo "Speck Round" 2
+		"hardpoint sprite" "hardpoint/speckle turret"
+		"hardpoint offset" 6
+		"fire effect" "speckle fire"
+		"inaccuracy" 0.9
+		"velocity" 60
+		"reload" 20
+		"burst reload" 15
+		"burst count" 3
+		"cluster"
+		"turret turn" 3
+		"firing energy" 72
+		"firing heat" 60
+		"firing force" 24
+		"submunition" "Speck"
+			"offset" -3 0
+		"submunition" "Speck"
+			"offset" 3 0
+	description "Designed for mid-sized ships, this variation on the Speckle Coilgun mounts a pair of coilguns onto a nimble turret. Its compact size allows even smaller ships to fit one, making it a versatile option for mid-ranged combat."
+	description "Of course, the inherently limited range and velocity of kinetic weapons makes the Speckle Turret a poor match for some of the longer ranged weapons used by the Avgi, but the sheer firepower it can put out makes it deadly in close range."
+
+
+outfit "Speck"
+	weapon
+		sprite "projectile/speck"
+			"frame rate" 60
+			"no repeat"
+		"hit effect" "bullet impact"
+		"lifetime" 15
+		"random lifetime" 3
+		"shield damage" 24
+		"hull damage" 48
+		"hit force" 12
+
+
+effect "speckle fire"
+	sprite "effect/emp torpedo fire"
+		"scale" 0.75
+		"random start frame"
+		"frame rate" 60
+	"lifetime" 3
+	"random angle" 30
+
+
+# Energy Weapons:
+
+outfit "Green Optical Laser"
+	category "Turrets"
+	cost 120000
+	thumbnail "outfit/avgi green laser"
+	"mass" 6
+	"outfit space" -6
+	"weapon capacity" -6
+	"turret mounts" -1
+	"optical jamming" 6
+	"lasing power" -1
+	weapon
+		"hardpoint sprite" "hardpoint/avgi green laser"
+		sound "disruptor"
+		"die effect" "laser ejecta" 4
+		"die effect" "green laser reflection" 30
+		"die effect" "green laser scattering"
+		"fire effect" "green laser fire"
+		"anti-missile" 2
+		"velocity" 1800
+		"lifetime" 1
+		"firing energy" 9
+		"firing heat" 9
+	description "This lightweight turret can be fed a beam of light from a lasing generator, which is then combined and focused to produce an intense beam suitable for targeting and destroying missiles. Unlike a typical pulsed-shot human antimissile system, this turret seems designed to hold a beam on target at an extreme distance for a longer period of time using its precise optics, making it better suited for defending and supporting other ships from afar."
+	description "The turret itself only contains the optics needed to synthesize, focus, and target the beam, allowing the laser light to be produced elsewhere. Because of this, an Optical Lasing Generator is needed to run this turret, providing enough power to support three Green Lasers."
```

Comment:
```suggestion
	description "The turret itself only contains the optics needed to synthesize, focus, and target the beam, requiring the laser light to be produced elsewhere. Because of this, an Optical Lasing Generator is needed to run this turret, providing enough power to support three Green Lasers."
```